woocommerce/woocommerce-gutenberg-products-block
# Move DropdownSelect to `@woocommerce/blocks-checkout` package

This PR will move the `DropdownSelect` component to the `@woocommerce/blocks-checkout` package. This will make it easier for third parties to re-use some of the UI we have built.

This is especially relevant for 75-gh-woocommerce/woocommerce-cart-add-ons


Helps 75-gh-woocommerce/woocommerce-cart-add-ons




### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Add a `Filter by Attribute` block and a `All Products` block and ensure they both load in the editor.
2. Choose an attribute and then in the Block Settings, set the display style to `Dropdpown`
<img src="https://user-images.githubusercontent.com/5656702/149150775-e7689e24-9a73-4c94-bd76-639d4af24a98.png" width="300"/>

3. View the block on the front end and ensure the dropdown works correctly and filters the products shown in the `All Products` block.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).

### Changelog

> Add suggested changelog entry here.
